src/rust/Dockerfile

```diff
@@ -1,21 +1,27 @@
+# syntax=docker/dockerfile:1.3-labs
+# We use the above syntax for here documents:
+# https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#user-content-here-documents
+
 FROM rust:1-slim-bullseye AS base
 
 ARG RUST_BUILD=debug
 
-SHELL ["/bin/bash", "-c"]
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Necessary for rdkafka; pkg-config & libssl-dev for cargo-tarpaulin build
 #
 # Ignore this lint about deleteing the apt-get lists (we're caching!)
-# hadolint ignore=DL3009
+# hadolint ignore=DL3009,SC1089
```

Comment:
Just based on 
https://github.com/koalaman/shellcheck/wiki/SC1089
i'm surprised this is triggering

Reply:
Yeah, I thought that was weird. 🤷

Reply:
Yeah, this is confusing. 🤔